### DISCUSSION TITLE
卸载u盘成功应该有提示

### DISCUSSION THREAD

**Discussion by widon1104** [linuxdeepin/developer-center#2968] [Features Request & Ideas | 特性请求 & 头脑风暴]
## 缺陷描述

现在卸载u盘，如果失败是有提示的，但是我觉得如果卸载成功应该也有有一个提示，比如说“现在u盘已经可以安全的移出了”，有这个提示这样用户更加安心

### 发行版以及版本

deepin15.11

**Comment by honyxiao** [comment]
+1，最好用不同的弹窗颜色加以区分。
早几天正好同事复制文件，还在sync的时候，就卸载U盘，结果弹出一串失败的提示（还是英文的）
同事以为是成功的消息，就拔了，结果数据没了。。。

**Comment by XMuli** [comment]
已经向团队提出了此建议

**Comment by meiyixiang** [comment]
你好你的建议在现在最新版本已经支持，卸载失败或卸载成功在通知栏都会有提示，请升级至最新的20.5版本
